File: bronze/30802/python/main.py

if (__name__ == "__main__"):
    n = int(input())
    size = list(map(int, input().split()))
    t, p = map(int, input().split())
    count = 0
    for i in range(6):
        if (size[i] % t == 0):
            count += int(size[i] / t)
        else:
            count += int(size[i] / t) +1
    
    print(f"{count}\n{int(n / p)} {n % p}")

File: bronze/30802/c++/main.cpp
#include <iostream>

int main()
{
    int n, size[6], t, p, count = 0;

    std::cin >> n;
    for (int _ = 0; _ < 6; _++) {
        std::cin >> size[_];
    }
    std::cin >> t >> p;

    for (int i = 0; i < 6; i++) {
        if (size[i] % t == 0) {
            count += (size[i] / t);
        } else {
            count += (size[i] / t) +1;
        }
    }

    std::cout << count << '\n' << (n / p) << ' ' << (n % p);

    return 0;
}